cranelift/module/src/backend.rs

```diff
@@ -179,6 +179,10 @@ where
 pub fn default_libcall_names() -> Box<dyn Fn(ir::LibCall) -> String> {
     Box::new(move |libcall| match libcall {
         ir::LibCall::Probestack => "__cranelift_probestack".to_owned(),
+        ir::LibCall::UdivI64 => "udiv".to_owned(),
+        ir::LibCall::SdivI64 => "sdiv".to_owned(),
+        ir::LibCall::UremI64 => "urem".to_owned(),
+        ir::LibCall::SremI64 => "srem".to_owned(),
```

Comment:
Please use the same names as compiler-builtins. For example `__udivdi3` for `UdivI64`:

https://github.com/rust-lang/compiler-builtins/blob/1108d2af9d9df3ae14b06d35a11190aa55097ab8/src/int/udiv.rs#L232

Reply:
Will do, I wasn't sure which convention cranelift was using here.

Reply:
Except for `__cranelift_probestack` all functions here are defined by system libraries.

Reply:
What's the implementation in `runtime/src/libcalls.rs` for, then?

Reply:
That one is used for Wasmtime only. This is a default mapping for other users of Cranelift.

Reply:
Done